src/web_app/js/appcontroller.js

```diff
@@ -519,9 +519,9 @@ AppController.prototype.loadUrlParams_ = function() {
   this.loadingParams_.opusStereo = urlParams['stereo'];
   this.loadingParams_.videoSendBitrate = urlParams['vsbr'];
   this.loadingParams_.videoSendInitialBitrate = urlParams['vsibr'];
-  this.loadingParams_.videoSendCodec = urlParams['vsc'];
+  this.loadingParams_.videoSendCodec = urlParams['vsc'] || 'VP9';
```

Comment:
I tend to think that we should only do this for the receive codec. IOW, we should just change the outgoing offer to promote VP9, we shouldn't clobber the incoming offer.

Having a DEFAULT_VIDEO_CODEC constant rather than hardcoding 'VP9' would also be a good idea.